lib/config/error-codes/api.json

```diff
@@ -106,12 +106,12 @@
         },
         "creation_failed": {
           "code": 6,
-          "message": "Some document creations failed : %s.",
+          "message": "Some document creations failed.",
```

Comment:
Note: obsoleted by #1481
(same for the other message change below)

Reply:
I will wait for https://github.com/kuzzleio/kuzzle/pull/1481 to be merged

Reply:
The PartialError for mXXX routes have been removed, so I removed the associated errors